docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,403 @@
+      name: {metric_name}                               # Required. Example: "FPR wrt class 0 restricted to feature gender:0 and age:21".
+      dtype: {metric_dtype}                             # Required. Example: "float".
+      value: {metric_value}                             # Required. Example: 0.75.
+      labels:
+        - name: {label_name}                            # Optional. Example: "gender".
+          type: {label_type}                            # Optional. Exmple "feature".
+          dtype: {label_type}                           # Optional. Example: "string".
+          value: {label_value}                          # Optional. Example: "female".
+    measurements:
+      # Bar plots should be able to capture SHAP and Robustness Toolbox from AI Verify.
+      bar_plots:
+      - type: {measurement_type}                        # Required. Example: "SHAP".
+        name: {measurement_name}                        # Optional. Example: "Mean Absolute Shap Values".
+        results:
+        - name: {bar_name}                              # Required. The name of a bar.
+          value: {bar_value}                            # Required. The corresponding value.
+      # Graph plots should be able to capture graph based measurements such as partial dependence and accumalated local effect.
+      graph_plots:
+      - type: {measurement_type}                        # Required. Example: "partial_dependence".
+        name: {measurement_name}                        # Optional. Example: "Partial Dependence Plot".
+        # Results store the graph plot data. So far all plots are depenendend on a combination of a specific class (sometimes) and feature (always).
+        # For example partial dependence plots are made for each feature and class.
+        results:
+         - class: {class_name}                          # Optional. Name of the output class the graph depends on.
+           feature: {feature_name}                      # Required. Name of the feature the graph depends on.
+           data:
+            - x_value: {x_value}                        # Required. The x value of the graph data.
+              y_value: {y_value}                        # Required. The y value of the graph data.
+
+```
+
+### Assessment Card
+
+```yaml
+name: {assessment_name}                               # Required. Example: IAMA.
+date: {assessment_date}                               # Required. Example: 25-03-2025.
+contents:
+  - question: {question_text}                         # Required. Example: "Question 1: ...".
+    answer: {answer_text}                             # Required. Example: "Answer: ...".
+    remarks: {remarks_text}                           # Optional. Example: "Remarks: ...".
+    authors:                                          # Optional. Example: "['John', 'Peter']".
+      - name: {author_name}
+    timestamp: {timestamp}                            # Optional. Example: 1711630721.
+````
```

Comment:
```suggestion
```
```

Reply:
@ChristopherSpelt here I tried to suggest the fix to go from 4x grave to 3x grave